### PR TITLE
docs: document missing configuration

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

**Review by kkujawa-sumo — APPROVED** [review]

**Review by rnishtala-sumo — APPROVED** [review]

File: ci/check_configuration_keys.py [hunk 1]
```diff
@@ -28,30 +76,38 @@ def main(values_path: str, readme_path: str) -> None:
...
+        print('*' * 20)
+        print(f'Keys in readme not existing in values ({len(readme_distinct)}):')
+        print('*' * 20)
+        for key in readme_distinct:
+            print(key)
+
+    if diff_defaults:
+        max_key_length = max(len(key) for key in diff_defaults.keys())
+
+        print('*' * 20)
```
Comment (rnishtala-sumo) [inline ci/check_configuration_keys.py:96]: should there be a logger for this?

File: ci/check_configuration_keys.py [hunk 2]
```diff
@@ -28,30 +76,38 @@ def main(values_path: str, readme_path: str) -> None:
...
+        if not full_diff:
+            print(f'| {"Key":{max_key_length}} | {"Default for readme":100} | {"Default for values":100} |')
+            print(f'|{"-"*(max_key_length+2)}|{"-"*102}|{"-"*102}|')
+
+        for key in sorted(diff_defaults.keys()):
+            readme_value, values_value = diff_defaults[key]
+
+            # Show only first 100 characters of every default
+            if full_diff:
```
Comment (rnishtala-sumo) [inline ci/check_configuration_keys.py:107]: should we check for this outside the loop?

File: ci/check_configuration_keys.py (hunk 1 above)
Comment (sumo-drosiek) [reply ci/check_configuration_keys.py:96]: I consider it as output, not logs

File: ci/check_configuration_keys.py (hunk 2 above)
Comment (sumo-drosiek) [reply ci/check_configuration_keys.py:107]: `readme_value` and `values_value` is assigned in the loop. Alternatively I could create two independent loops on `for key in sorted(diff_defaults.keys()):` level. Not sure which solution is better

Comment (sumo-drosiek) [reply ci/check_configuration_keys.py]: Changed it and it seems to be better in terms of readability :see_no_evil:

**Review by swiatekm-sumo — APPROVED** [review]
The script changes look fine to me, can't really review the README ones.

**Comment by sumo-drosiek** [comment]
@swiatekm-sumo Actually this should be helpful for README review:

https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2739/files/1b1642ec3964f12d2e68bb0f03d4c030c505a9b3..0cb9149ee594ae5e4dab8a90093804d4ad07ac32?diff=split&w=0

**Review by swiatekm-sumo — COMMENTED** [review]
Found some misspellings in the README

File: deploy/helm/sumologic/README.md [hunk 3]
```diff
@@ -106,18 +118,21 @@ The following table lists the configurable parameters of the Sumo Logic chart an
...
+| `sumologic.setup.job.resources`                                                  | Resource requests and limits for the rthe setup Job.                                                                                                                                                                                                                                                                                                                 | `{"limits": {"memory": "256Mi", "cpu": "2000m"}, "requests": {"memory": "64Mi", "cpu": "200m"}}`                                                                                     |
```
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply deploy/helm/sumologic/README.md:129]:
```suggestion
| `sumologic.setup.job.resources`                                                  | Resource requests and limits for the setup Job.                                                                                                                                                                                                                                                                                                                 | `{"limits": {"memory": "256Mi", "cpu": "2000m"}, "requests": {"memory": "64Mi", "cpu": "200m"}}`                                                                                     |
```

**Pull request merged**